backend/API/lambda_function.py

```diff
@@ -0,0 +1,20 @@
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+import nltk
+import json
+
+def lambda_handler(event, context):
+    # add temp directory to data path
+    nltk.data.path.append('/tmp')
+    
+    # download data to temp directory
+    nltk.download('vader_lexicon', download_dir='/tmp')
+    
+    # execute sentiment analyzer 
+    sid = SentimentIntensityAnalyzer()
+    ss = sid.polarity_scores(event['feedback'])
+    out = {'neg': ss['neg'], 'neu': ss['neu'], 'pos': ss['pos']}
+
+    return {
+        'statusCode': 200,
+        'body': json.dumps(out)
```

Comment:
try to return `'body': out`. This might get rid of the extra backslashes around the output json. It's probably due to dictionary being double encoded with `json.dumps()`, I ran into the same behavior as well.

Reply:
Good catch, this corrected the out. Updated the code base and Lambda function.